### PR TITLE
try to fix travis

### PR DESCRIPTION
**Pull request by k-okada**
- changeing .travis.yml for current environment
- but we need to fix some of test program to pass tests

### CODE REVIEW

**Comment by k-okada** [comment]
ready for review and merge , specially please check 
https://github.com/start-jsk/rtmros_hironx/pull/458/commits/ac2e40285d9062aec90208540192ed41dad64614
https://github.com/start-jsk/rtmros_hironx/pull/458/commits/12dbe4ed531ade2f75c1354a2b5b1e966a7d248e
https://github.com/start-jsk/rtmros_hironx/pull/458/commits/c3f875fc87ef82030c02740c4523e64df74dde99
https://github.com/start-jsk/rtmros_hironx/pull/458/commits/69be9cb9e4bbce3cb6a4b2621e4aadc6c1e31cb4
which including significant change on source tree.

File: .travis.yml [hunk 1]
```diff
@@ -83,13 +83,15 @@ script: # All commands must exit with code 0 on success. Anything else is consid
   - if [ $BUILDER == catkin ]; then export TARGET_PKG=`find build*/*hiro* -name Makefile -print |  sed s@.*/\\\\\([^\/]*\\\\\)/Makefile@\\\1@g` ; fi
   # - if [ $BUILDER == catkin ]; then source devel/setup.sh; export EXIT_STATUS=0; for pkg in $TARGET_PKG; do (roscd $pkg; [ "`find . -iname '*.test'`" == "" ] && echo "[$pkg] No tests were found!!!"  || find . -iname "*.test" -print0 | xargs -0 -n1 rostest || export EXIT_STATUS=$?;) done; [ $EXIT_STATUS == 0 ] ; fi
   - export EXIT_STATUS=$?
+  - if [ $BUILDER == catkin ]; then source devel*/setup.bash              ; fi
```
Comment (130s) [inline .travis.yml]: Assuming the asterisk (devel`*`) is to cover both `devel` and `devel_isolated` spaces.

File: hironx_moveit_config/test/test-hironx-moveit.test [hunk 2]
```diff
@@ -15,5 +15,5 @@
     <arg name="load_robot_description" value="true"/>
   </include>
 
-  <test type="test_hironx_moveit.py" pkg="hironx_ros_bridge" test-name="test_hironx_moveit" time-limit="30" retry="4" />
+  <test type="test_hironx_moveit.py" pkg="hironx_moveit_config" test-name="test_hironx_moveit" time-limit="30" retry="4" />
```
Comment (130s) [inline hironx_moveit_config/test/test-hironx-moveit.test]: +1

File: hironx_moveit_config/test/test_hironx_moveit.py [hunk 3]
```diff
@@ -58,7 +58,6 @@ def __init__(self, *args, **kwargs):
     @classmethod
     def setUpClass(self):
 
-        rospy.init_node("test_hironx_moveit")
```
Comment (130s) [inline hironx_moveit_config/test/test_hironx_moveit.py:61]: I'm not sure why removing this line solves the exception "`rospy.init_node() has already been called with different arguments`".

File: hironx_moveit_config/test/test_hironx_moveit.py [hunk 4]
```diff
@@ -74,8 +74,8 @@ def setUpClass(self):
         self.init_rtm_jointvals = [0.010471975511965976, 0.0, -1.7453292519943295, -0.26529004630313807, 0.16406094968746698, -0.05585053606381855,
                                    -0.010471975511965976, 0.0, -1.7453292519943295, 0.26529004630313807, 0.16406094968746698, 0.05585053606381855]
 
-        self.init_rtm_jointvals_factory = [0.010471975511965976, 0.0, -1.7453292519943295, -0.26529004630313807, 0.16406094968746698, -0.05585053606381855,
-                                           -0.010471975511965976, 0.0, -1.7453292519943295, 0.26529004630313807, 0.16406094968746698, 0.05585053606381855]
+        self.init_rtm_jointvals_factory = [-1.3877787807814457e-17, 1.0842021724855044e-19, -2.2689280275926285, -4.440892098500626e-16, -2.220446049250313e-16, 0.0,
+                                           0.0, 1.0842021724855044e-19, -2.2689280275926285, 2.220446049250313e-16, -1.1102230246251565e-16, 5.551115123125783e-17]
```
Comment (130s) [inline hironx_moveit_config/test/test_hironx_moveit.py:78]: +1

File: .travis.yml [hunk 5]
```diff
@@ -77,7 +77,7 @@ before_script: # Use this to prepare your build for testing e.g. copy database c
 script: # All commands must exit with code 0 on success. Anything else is considered failure.
   # for catkin
   - if [ $BUILDER == catkin -a $USE_DEB == true  ]; then catkin_make ${ROS_PARALLEL_JOBS}            ; fi
-  - if [ $BUILDER == catkin -a $USE_DEB == false ]; then mkdir -p $HOME/ros/ws_$REPOSITORY_NAME/devel_isolated/hrpsys/; echo "$HOME/ros/ws_$REPOSITORY_NAME/devel_isolated/hrpsys/" > $HOME/ros/ws_$REPOSITORY_NAME/devel_isolated/hrpsys/.catkin; fi
+  # - if [ $BUILDER == catkin -a $USE_DEB == false ]; then mkdir -p $HOME/ros/ws_$REPOSITORY_NAME/devel_isolated/hrpsys/; echo "$HOME/ros/ws_$REPOSITORY_NAME/devel_isolated/hrpsys/" > $HOME/ros/ws_$REPOSITORY_NAME/devel_isolated/hrpsys/.catkin; fi
```
Comment (130s) [inline .travis.yml:80]: Not a big deal but we might as well remove the line instead of commenting out?

**Comment by 130s** [comment]
Thank you very much. I don't fully understand all the changes but Travis seems happy.
+1 overall.

File: hironx_moveit_config/test/test_hironx_moveit.py (hunk 3 above)
Comment (k-okada) [inline hironx_moveit_config/test/test_hironx_moveit.py:61]: On Thu, May 5, 2016 at 1:25 AM, Isaac I.Y. Saito notifications@github.com
wrote:

> I'm not sure why removing this line solves the exception
> "rospy.init_node() has already been called with different arguments".

ROS_Clinet calls rospy.init_node()
https://github.com/start-jsk/rtmros_hironx/blob/indigo-devel/hironx_ros_bridge/src/hironx_ros_bridge/ros_client.py#L94
,

## 

◉ Kei Okada

Comment (130s) [inline hironx_moveit_config/test/test_hironx_moveit.py:61]: I see, [ROS_Client](https://github.com/start-jsk/rtmros_hironx/pull/458/files/12dbe4ed531ade2f75c1354a2b5b1e966a7d248e..c3f875fc87ef82030c02740c4523e64df74dde99#diff-68a1160ea0ff512aa6963f8383742dd4L62) is also a ROS node.

**Pull request merged**